west.yml

```diff
@@ -31,14 +31,17 @@ manifest:
       url-base: https://github.com/ARMmbed
     - name: civetweb
       url-base: https://github.com/civetweb
+    - name: haam
+      url-base: https://github.com/hakonfam
 
   # The list of external projects for the nRF Connect SDK.
   #
   projects:
     - name: zephyr
       repo-path: fw-nrfconnect-zephyr
       west-commands: scripts/west-commands.yml
-      revision: v2.1.99-ncs1-rc1
+      revision: pm-ram
```

Comment:
If you open a draft PR to fw-nrfconnect-zephyr instead, we can review.

Reply:
will do